.github/workflows/ci.yml

```diff
@@ -14,21 +14,31 @@ jobs:
         target:
           - os: linux
             cpu: amd64
+          - os: linux-gcc-14 # This is to use ubuntu 24 and install gcc 14. Should be removed when ubuntu-latest is 26.04
+            cpu: amd64
           - os: linux
             cpu: i386
           - os: macos
             cpu: amd64
+          - os: macos
+            cpu: arm64
           - os: windows
             cpu: amd64
-          #- os: windows
-            #cpu: i386
-        branch: [version-1-6, version-2-0, devel]
+        branch: [version-2-0, version-2-2, devel]
```

Comment:
as with the other low-level libraries, this should test 1.6 as well (in particular, it should test what the nimble file specifies as compatible)